src/cmdstan-guide/pathfinder_config.Rmd

```diff
@@ -0,0 +1,162 @@
+reported at the beginning of the pathfinder output file as CSV comments.
+When the example model `bernoulli.stan` is run with `method=pathfinder`
+via the command line with all default arguments,
+the resulting Stan CSV file header comments show the complete set
+of default configuration options:
+```
+method = pathfinder
+  pathfinder
+    init_alpha = 0.001 (Default)
+    tol_obj = 9.9999999999999998e-13 (Default)
+    tol_rel_obj = 10000 (Default)
+    tol_grad = 1e-08 (Default)
+    tol_rel_grad = 10000000 (Default)
+    tol_param = 1e-08 (Default)
+    history_size = 5 (Default)
+    num_psis_draws = 1000 (Default)
+    num_paths = 4 (Default)
+    max_lbfgs_iters = 1000 (Default)
+    num_draws = 1000 (Default)
+    num_elbo_draws = 25 (Default)
+```
+
+The arguments `init_alpha` through `history_size` are used by the L-BFGS algorithm.
+The remaining arguments are used by the Pathfinder algorithm.
+
+## Pathfinder Configuration
+
+- `num_psis_draws` - Number of draws from PSIS sample (multi-path pathfinder).
+Must be a positive integer.  Default value is $1000$.
+
+- `num_paths` - Number of single pathfinders.
+Must be a positive integer.  Default value is $4$.
+
+- `max_lbfgs_iters` - Maximum number of L-BFGS iterations.
+Must be a positive integer.  Default value is $1000$.
+
+- `num_draws` - Number of approximate posterior draws for each single pathfinder.
+Must be a positive integer.  Default value is $1000$.
+
+- `num_elbo_draws` - Number of Monte Carlo draws to evaluate ELBO.
+Must be a positive integer.  Default value is $25$.
+
+
+## L-BFGS Configuration
+
+These areguments have the same defaults as for [optimization](#optimize_config).
+
+## Output CSV files
+
+For a single run of the Pathfinder algorithm, where argument `num_paths` is $1$,
+the default output is a single CSV file, `output.csv`.
+
+If `num_paths` is not specified, the default is to run $4$ single-path pathfinders.
+In this case,  Pathfinder output is written into a set of StanCSV files; one file which contains the importance resampled draws
+from multi-path Pathfinder, and one file which contains the output of each single-path Pathfinder run.
+Given an output file name, CmdStan adds suffixes to the base filename to distinguish between the output files.
+For the default output file name `output.csv`, the resulting set of CSV files are
+```
+output_pathfinder.csv
+output_1.csv
+output_2.csv
+output_3.csv
+output_4.csv
+```
+
+The CSV files sample files have the following structure:
+
+- The full set of configuration options available for the `pathfinder` method is
+reported at the beginning of the sampler output file as CSV comments.
+
+- The CSV header row consists of columns `lp_approx__`, `lp__`, and the Stan model parameters, transformed parameters, and generated quantities.
+
+- The data rows contain the corresponding values from either the PSIS sample (multi-path) or the single pathfinder sample.
```

Comment:
I'd drop this or say each row represents a draw (and hence the whole file represents a sample).